### PR TITLE
Add ServerBuilder#serverListener

### PR DESCRIPTION
**Pull request by imasahiro**

### CODE REVIEW

File: core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java [hunk 1]
```diff
@@ -443,11 +444,18 @@ public ServerBuilder defaultVirtualHost(VirtualHost defaultVirtualHost) {
         return this;
     }
 
+    /**
+     * Adds the {@link ServerListener}.
+     */
+    public ServerBuilder serverListner(ServerListener serverListener) {
```
Comment (anuraaga) [inline core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java]: serverListener (method name)

File: core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java [hunk 2]
```diff
@@ -57,7 +57,7 @@
  * // Build a server.
  * Server s = sb.build();
  * }</pre>
- * 
+ *
```
Comment (trustin) [inline core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java]: Thanks!

File: core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java (hunk 1 above)
Comment (trustin) [inline core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java]: the -> the specified

File: core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java [hunk 3]
```diff
@@ -443,11 +444,18 @@ public ServerBuilder defaultVirtualHost(VirtualHost defaultVirtualHost) {
         return this;
     }
 
+    /**
+     * Adds the {@link ServerListener}.
+     */
+    public ServerBuilder serverListner(ServerListener serverListener) {
+        serverListeners.add(serverListener);
```
Comment (trustin) [inline core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java]: `requireNonNull(serverListener, "serverListener")`

**Comment by trustin** [comment]
Merged. Thanks, @imasahiro !

**Pull request merged**